### PR TITLE
docs: add documentation about Unified Linux Client

### PR DESCRIPTION
**Pull request by joergsteffens**
- Add documentation about the Unified Linux Client.
- Updating the install section (URLs and Linux Distributions). Explaining EL
- Fix some MacOS download information

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

##### General

- [x] PR name is meaningful
- [x] Purpose of the PR is understood
- [x] Separate commit for this PR in the CHANGELOG.md, PR number referenced is same
- [x] Commit descriptions are understandable and well formatted

##### Source code quality

- [x] Required documentation changes are present and part of the PR
- [x] `bareos-check-sources --since-merge` does not report any problems
- [x] `git status` should not report modifications in the source tree after building and testing

### CODE REVIEW

File: docs/manuals/source/Appendix/OperatingSystems.rst [hunk 1]
```diff
@@ -54,29 +54,47 @@ Additionally, the following table gives an overview on the available package ver
...
 
-.. include:: /include/autogenerated/bareos-packages-table-redhat.rst.inc
+.. note::
 
-.. include:: /include/autogenerated/bareos-packages-table-fedora.rst.inc
+   The **Universal Linux Client** depends on the **OpenSSL** library
+   of the host in order to utilize security updates for this library.
 
-.. include:: /include/autogenerated/bareos-packages-table-suse.rst.inc
+It incorperates all functionality for normal backup and restore operations,
```
Comment (pstorz) [inline docs/manuals/source/Appendix/OperatingSystems.rst:76]:
```suggestion
It incorporates all functionality for normal backup and restore operations,
```

File: docs/manuals/source/Appendix/OperatingSystems.rst [hunk 2]
```diff
@@ -54,29 +54,47 @@ Additionally, the following table gives an overview on the available package ver
...
 
-.. include:: /include/autogenerated/bareos-packages-table-opensuse.rst.inc
+Currently it is provided as a Debian package.
+However, it it planed to provide it also in other formats.
 
-.. include:: /include/autogenerated/bareos-packages-table-debian.rst.inc
+The ULC components come in extra repositories (names starting with **ULC_**)
+at https://download.bareos.org/bareos/release/
+and consists of only a single package per platform.
+Therfore you can either add the repository to your system
```
Comment (pstorz) [inline docs/manuals/source/Appendix/OperatingSystems.rst:85]:
```suggestion
Therefore you can either add the repository to your system
```

File: docs/manuals/source/Appendix/OperatingSystems.rst [hunk 3]
```diff
@@ -54,29 +54,47 @@ Additionally, the following table gives an overview on the available package ver
...
+however it has only limited plugin support.
 
-.. include:: /include/autogenerated/bareos-packages-table-opensuse.rst.inc
+Currently it is provided as a Debian package.
+However, it it planed to provide it also in other formats.
 
-.. include:: /include/autogenerated/bareos-packages-table-debian.rst.inc
+The ULC components come in extra repositories (names starting with **ULC_**)
+at https://download.bareos.org/bareos/release/
+and consists of only a single package per platform.
```
Comment (pstorz) [inline docs/manuals/source/Appendix/OperatingSystems.rst:84]: This is not exactly true, as we also have the debug package (see  contents of https://download.bareos.org/bareos/experimental/nightly/ULC_deb_OpenSSL_1.1/amd64/)

File: docs/manuals/source/Appendix/OperatingSystems.rst [hunk 4]
```diff
@@ -54,29 +54,47 @@ Additionally, the following table gives an overview on the available package ver
...
-.. include:: /include/autogenerated/bareos-packages-table-ubuntu.rst.inc
 
-.. include:: /include/autogenerated/bareos-packages-table-univention.rst.inc
+Feature overview:
+
+  * Single package installation
+  * Repository based installation
+  * Minimal dependencies to system libraries (except OpenSSL)
+  * Uses host OpenSSL library
+  * Replacable by normal |fd|. No configuration change required.
```
Comment (pstorz) [inline docs/manuals/source/Appendix/OperatingSystems.rst:97]: I don't think that people understand what this means, and as it is a very good feature we should describe what it means. Also, people should use this mechanism to switch to the regular repository when for a new OS packages are available from Bareos.

File: docs/manuals/source/Appendix/OperatingSystems.rst [hunk 5]
```diff
@@ -54,29 +54,47 @@ Additionally, the following table gives an overview on the available package ver
...
-For general information about the packages, see :ref:`section-BareosPackages`.
+The Universal Linux Client is a |fd|,
+built in a way to have minimal dependencies to other libraries.
 
-.. include:: /include/autogenerated/bareos-packages-table-redhat.rst.inc
+.. note::
 
-.. include:: /include/autogenerated/bareos-packages-table-fedora.rst.inc
+   The **Universal Linux Client** depends on the **OpenSSL** library
+   of the host in order to utilize security updates for this library.
```
Comment (pstorz) [inline docs/manuals/source/Appendix/OperatingSystems.rst:74]: We should definitely put a warning, that ULC is intended to be ONLY used on OS that we do not have regular clients for.
All users with an OS that has OS-specific packages must use these packages.

**Review by pstorz — CHANGES_REQUESTED** [review]
I would suggest to remove the whole table regading the available versions.
We do now have the pkglists in the docs directories where the list of packages is documented and automatically verified. (See https://github.com/bareos/bareos/tree/master/docs/pkglists). I would suggest we refer to these lists.

File: docs/manuals/source/Appendix/OperatingSystems.rst (hunk 3 above)
Comment (joergsteffens) [reply docs/manuals/source/Appendix/OperatingSystems.rst]: Add remark about debug packages.

File: docs/manuals/source/Appendix/OperatingSystems.rst (hunk 4 above)
Comment (joergsteffens) [reply docs/manuals/source/Appendix/OperatingSystems.rst]: Added section.

File: docs/manuals/source/Appendix/OperatingSystems.rst (hunk 5 above)
Comment (joergsteffens) [reply docs/manuals/source/Appendix/OperatingSystems.rst:74]: Added warning.

**Comment by joergsteffens** [comment]
I tried to incorporate your feedback. Feel free to change this proposed section by adding own commits.

**Review by pstorz — APPROVED** [review]
Good work.  I had to add the CHANGELOG.md update and also two files in .gitingore for the universal client were missing.

**Pull request merged**